src/import/common.rs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::ops::Deref;
+use std::sync::Arc;
+
+use kvproto::metapb::*;
+use kvproto::kvrpcpb::*;
+use kvproto::importpb::*;
+
+use pd::RegionInfo;
+
+use super::client::*;
+
+// Just used as a mark, don't use them in comparison.
+pub const RANGE_MIN: &[u8] = &[];
```

Comment:
actually RANGE_MIN is used in comparison but RANGE_MAX is not?

Reply:
Well, just use them to express that a key is a `RANGE_MIN` or a `RANGE_MAX`, just make it more clear.